masterlist.yaml

```diff
@@ -371,33 +360,6 @@ plugins:
           - '[Manual Cleaning Guide](https://afkmods.iguanadons.net/index.php?/topic/4110-manual-cleaning-skyrim-and-skyrim-se-master-files/)'
         condition: 'checksum("Dawnguard.esm", 6061bcff) or checksum("Dawnguard.esm", 1CF3CECB)'
     dirty:
-      - <<: *dirtyPlugin
-        crc: 0x051fcdc8
-        itm: 628
-        udr: 82
-        nav: 57
-      - <<: *dirtyPlugin
-        crc: 0x0d2cf128
-        itm: 6
-        nav: 57
-      - <<: *dirtyPlugin
-        crc: 0x1b95f26b
-        itm: 6
-        nav: 57
-      - <<: *dirtyPlugin
-        crc: 0x09ff5624
-        itm: 6
-        nav: 57
-      - <<: *dirtyPlugin
-        crc: 0x0d694ec0
-        itm: 621
-        udr: 82
-        nav: 57
-      - <<: *dirtyPlugin
-        crc: 0xbfba5a40
-        itm: 621
-        udr: 82
-        nav: 57
```

Comment:
I think these old cleaning info's are just noise at this point. 
There is no version listed, and updates to xEdit cleaning will make them inaccurate.